Packs/MicrosoftExchangeServer/ModelingRules/MicrosoftExchangeServerModeling/MicrosoftExchangeModeling.xif

```diff
@@ -0,0 +1,17 @@
+[MODEL: dataset="filestream_filestream_raw", model="Email"]
```

Comment:
need to change to dataset="msft_exchange_raw"

Reply:
Done